Algorithm.CSharp/BasicTemplateAlgorithm.cs

```diff
@@ -84,18 +84,18 @@ public override void OnData(Slice data)
             {"Drawdown", "2.200%"},
             {"Expectancy", "0"},
             {"Net Profit", "1.692%"},
-            {"Sharpe Ratio", "8.888"},
-            {"Probabilistic Sharpe Ratio", "67.609%"},
+            {"Sharpe Ratio", "7.01"},
+            {"Probabilistic Sharpe Ratio", "68.391%"},
             {"Loss Rate", "0%"},
             {"Win Rate", "0%"},
             {"Profit-Loss Ratio", "0"},
-            {"Alpha", "-0.005"},
-            {"Beta", "0.996"},
-            {"Annual Standard Deviation", "0.222"},
-            {"Annual Variance", "0.049"},
-            {"Information Ratio", "-14.565"},
-            {"Tracking Error", "0.001"},
-            {"Treynor Ratio", "1.978"},
+            {"Alpha", "0.885"},
+            {"Beta", "0.874"},
```

Comment:
Do we know why the beta is getting worse in these minute resolution cases?

Reply:
Has to do with the order being executed at market open.

First benchmark value is day 1 at 12AM, because we buy during day 1 at open, the price will have changed. Then the performance for day 1 benchmark and equity will be off.

Reply:
The commits below address this by skipping day 1 sample in the statistics builder